content/docs/04-clusters/01-public-cloud/03-azure.md

```diff
@@ -76,35 +76,44 @@ For this, we first need to create an Azure Active Directory (AAD) Application wh
 
 The following steps need to be performed to provision a new Azure cluster:
 
-1. Provide the basic cluster information like Name, Description, and Tags.
+1. Provide the basic cluster information like **Name**, **Description** (optional), and **Tags** (optional) and select the [**Azure Cloud Account**](/clusters/public-cloud/azure#creatinganazurecloudaccount) from the drop-down menu. Azure cloud accounts with credentials need to be pre-configured in project settings. Click **next** button.

```

Comment:
The last sentence is a prerequisite, so if this was structured into a How-to this would be addressed.